hack/make/deps.mk

```diff
@@ -8,11 +8,11 @@ KUBECTL_SUM_s390x ?= $(shell curl -L "https://dl.k8s.io/release/$(KUBECTL_VERSIO
 
 # renovate: datasource=github-release-attachments depName=kubernetes-sigs/kustomize
 KUSTOMIZE_VERSION := v5.3.0
-# renovate: datasource=github-release-attachments depName=kubernetes-sigs/kustomize digestVersion=v5.3.0
+# renovate: datasource=github-release-attachments depName=kubernetes-sigs/kustomize digestVersion=kustomize/v5.3.0
```

Comment:
My only concern/thought about why this might not work is if `/` won't get us the correct result. The URL in browser uses the encoded `%2F` in place of `/`. So maybe we need to put that here too?

Reply:
Oh - actually it should be fine I guess. Looks like this URL loads correctly too: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize/v5.3.0